src/main/java/com/almostreliable/summoningrituals/altar/IndestructibleAltarBlock.java

```diff
@@ -0,0 +1,10 @@
+package com.almostreliable.summoningrituals.altar;
+
+import net.minecraft.world.level.block.SoundType;
+import net.minecraft.world.level.material.Material;
+
+public class IndestructibleAltarBlock extends AltarBlock{
+    public IndestructibleAltarBlock() {
+        super(Properties.of(Material.STONE).strength(-1.0F, 3600000.0F).sound(SoundType.STONE));
+    }
+}
```

Comment:
This should not be another class. You can make use of the `AltarBlock` class with your new constructor. Remove this class and pass the block properties directly from `Setup`.